Sources/CombineRextensions/NavigationLink+Extensions.swift

```diff
@@ -61,7 +66,7 @@ extension NavigationLink {
             self.init(
                 destination: producer.view(secondScreen),
                 tag: secondScreen,
-                selection: store.binding[path] { value in
```

Comment:
This fix is just to forward file, line, function and info to the Dispatcher properly.